runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -171,6 +171,9 @@ class INTERPRETER_CLASS
 	J9Method *_sendMethod;
 #if defined(DO_SINGLE_STEP)
 	bool _skipSingleStep;
+#endif
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
```suggestion
#if defined(DEBUG_VERSION) || defined(J9VM_OPT_CRIU_SUPPORT)
```
Unless the following checking `_dontSkipHooks` are decorated with `#if !defined(DEBUG_VERSION)`.